### PR TITLE
[DOCS-6517] AWS Lambda additions to CLI, CDK and Serverless plugin

### PR DESCRIPTION
**Pull request by aliciascott**
### What does this PR do? What is the motivation?
This PR adds additional steps for newly added capabilities for AWS Lambda for CLI, AWS CDK, and Serverless Plugin.

### Merge instructions

- Approved by PM
- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/security/application_security/enabling/serverless.md [hunk 1]
```diff
@@ -216,6 +216,136 @@ To install and configure the Datadog Serverless Framework plugin:
...
+
+**If you are configuring initial tracing for your functions**, perform the following steps:
+
+1. Install the Datadog CLI client:
+
+    ```sh
+    npm install -g @datadog/datadog-ci
+    ```
+
+2. If you are new to Datadog serverless tracing, launch the Datadog CLI in interactive mode to guide your first installation for a quick start, and ignore the remaining steps. To permanently install Datadog for your production applications, skip this step and follow the remaining steps to run the Datadog CLI command in your CI/CD pipelines _after_ your normal deployment.
```
Comment (hestonhoffman) [inline content/en/security/application_security/enabling/serverless.md:245]: This works, but it's an added cognitive load for the reader cos you're asking them to make a choice inside the step and the command they need to run if they choose the first option is split by the second option. It's kind of a tough one cos this section is so dense already. Have you thought about breaking this into sub sections? For example:
### If this is your first time...  
steps 1 and 2

### If you've used x before and want to...
steps 1,3,4 etc

File: content/en/security/application_security/enabling/serverless.md [hunk 2]
```diff
@@ -216,6 +216,136 @@ To install and configure the Datadog Serverless Framework plugin:
...
+
+    Datadog CLI requires access to the AWS Lambda service, and depends on the AWS JavaScript SDK to [resolve the credentials][1]. Ensure your AWS credentials are configured using the same method you would use when invoking the AWS CLI.
+
+4. Configure the Datadog site:
+
+    ```sh
+    export DATADOG_SITE="<DATADOG_SITE>"
+    ```
+
+    Replace `<DATADOG_SITE>` with {{< region-param key="dd_site" code="true" >}} (ensure the correct SITE is selected on the right).
```
Comment (hestonhoffman) [inline content/en/security/application_security/enabling/serverless.md:261]:
```suggestion
    Replace `<DATADOG_SITE>` with {{< region-param key="dd_site" code="true" >}} (ensure the correct **Datadog site** is selected on the right-hand side of this page).
```

**Review by hestonhoffman — APPROVED** [review]
LGTM! I left you some feedback on breaking up one of the steps, so feel free to re-request a review if you go that route and you want me to take a look

File: content/en/security/application_security/enabling/serverless.md (hunk 1 above)
Comment (aliciascott) [reply content/en/security/application_security/enabling/serverless.md:245]: Thanks @hestonhoffman that's great feedback and agree after re-reading this, it does get wordy and confusing, @sourabhkatti do you have some ideas how this should be re-structured a bit ?

Comment (sourabhkatti) [reply content/en/security/application_security/enabling/serverless.md:245]: This makes sense, we can keep just the "new to datadog..." flow and remove the "existing to datadog..." flow. 

If a customer already knows the steps, they can intuitively skip ahead anyway

**Pull request merged**